src/main/java/com/flagsmith/utils/ModelUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.flagsmith.utils;
+
+import com.flagsmith.flagengine.identities.traits.TraitModel;
+import com.flagsmith.models.SdkTraitModel;
+import com.flagsmith.models.TraitConfig;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class ModelUtils {
```

Comment:
nit: I'd prefer to see the `public` methods defined at the top of the class so it's easier to see what the external interface looks like.

Reply:
This is done.